packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -3,6 +3,16 @@ title: Okta API Products Release Notes 2021
 ---
 ## July
 
+### Weekly Release 2021.07.2
+
+| Change                                                                         | Expected in Preview Orgs |
+|--------------------------------------------------------------------------------|--------------------------|
+| [Bug fixed in 2021.07.2](#bug-fixed-in-2021-07-2)                            | July 28, 2021            |
+
+#### Bug fixed in 2021.07.2
+
+Authentication responses didn't include the Provider ID (`IdP.id`) or Provider Type (`IdP.type`) properties for some Okta-sourced users. (OKTA-410677)
```

Comment:
```suggestion
The [Get current session API](/docs/reference/api/sessions/#get-current-session) (`/api/v1/sessions/me`) didn't include the Provider ID (`IdP.id`) or Provider Type (`IdP.type`) properties for some Okta-sourced users. (OKTA-410677)
```

Reply:
I know this is already approved by the SME, but it would be nice to document the specific API where the the bug is occurring.
This is a suggestion, so I'm approving the PR if you decide to go with the original text. 🚢

Reply:
@brianduffield-okta please check Van's comment and see if there's anything you want to add. Thanks.

Reply:
Hi @vanngo-okta ... I originally wanted to add this, too; however, SMEs were insistent on keeping it generic as this bug happened outside the sessions object, as well. cc @christinechoy-okta